javascript-practice/src/styles/abstracts/_variables.scss

```diff
@@ -1,12 +1,17 @@
 $primary-font: 'Roboto';
 
 // Color
+$light-primary-color: #00bc2a;
 $primary-color: #2db84c;
+$dark-primary-color: #009421;
 $primary-text-color: #000;
 $secondary-text-color: #0000008a;
 $secondary-light-text-color: #00000042;
 $receive-money-color: #e51c23;
 $deducted-money-color: #039be5;
+$color-background-input: #f5f5f5;
```

Comment:
color variable not the same format naming?

Reply:
Fixed.